astroid/nodes/node_classes.py

```diff
@@ -80,6 +80,11 @@
     from astroid import nodes
     from astroid.nodes import LocalsDictNodeNG
 
+if sys.version_info >= (3, 8) or TYPE_CHECKING:
+    from functools import cached_property  # pylint: disable=ungrouped-imports
```

Comment:
```suggestion
    # pylint: disable-next=ungrouped-imports
    from functools import cached_property
```
This looks like something we should fix in pylint -> `ungrouped-imports` 🤔

Reply:
Yeah something weird is going on here..